frontend/src/graphql/queries.ts

```diff
@@ -15,3 +15,26 @@ const findAllUserProfiles = async () => {
   return data.users;
 };
 export default findAllUserProfiles;
+
+const joinTownMutation = gql`
+  mutation joinTown($TownJoinRequest: TownJoinRequest!)
+  @rest(type: "Town", path: "/sessions", method: "POST") {
+    isOK
+    response
```

Comment:
response {
      coveyUserID
      coveySessionToken
      
      providerVideoToken
      currentPlayers {
        _id
      }
      friendlyName
      isPubliclyListed
      
    }

Response should contain all of it , otherwise the calling function won't be able to access these fields.